httphandlers/http_handlers.go

```diff
@@ -186,6 +203,11 @@ func writeRequestLog(logger *logger.UPPLogger, req *http.Request, transactionID
 		"userAgent":      req.UserAgent(),
 	})
 
+	headers := getRequestHeaders(req, fh)
+	if len(headers) != 0 {
+		entry = entry.WithField("headers", headers)
```

Comment:
I completely fine with having "headers" key and logging all the headers as values of this key. I was just thinking around just logging each header as separate key as it's currently done for the `User-Agent` for example. It could be a bit easier query if you want to filter requests based on specific header.